source/guides/stacks/dlrs.rst

```diff
@@ -1079,15 +1079,15 @@ OpenVINO is a trademark of Intel Corporation or its subsidiaries
 
 .. _Getting Started with Kubeflow: https://github.intel.com/verticals/usecases/blob/56717f4642ecd958dc93bbc361c551dfc578d3ed/kubeflow/README.md#getting-started-with-kubeflow
 
-.. _Eigen: https://hub.docker.com/r/clearlinux/stacks-dlrs-oss/
+.. _Eigen: https://hub.docker.com/r/sysstacks/dlrs-tensorflow-clearlinux/
 
-.. _Intel MKL-DNN: https://hub.docker.com/r/clearlinux/stacks-dlrs-mkl/
+.. _Intel MKL-DNN: https://hub.docker.com/r/sysstacks/dlrs-tensorflow-clearlinux/
 
 .. _PyTorch with OpenBLAS: https://hub.docker.com/r/clearlinux/stacks-pytorch-oss
 
 .. _PyTorch with Intel MKL-DNN: https://hub.docker.com/r/clearlinux/stacks-pytorch-mkl
```

Comment:
nits:
TensorFlow Eigen:  sysstacks/dlrs-tensorflow-clearlinux:v0.6.0-oss
Tensorflow MKL-DNN: 
- 1.15 - sysstacks/dlrs-tensorflow-clearlinux:v0.6.0
- 2.2.0 - sysstacks/dlrs-tensorflow2-clearlinux:v0.6.0

There is only one version of Eigen based image for TF

PyTorch with OpenBLAS - sysstacks/dlrs-pytorch-clearlinux:v0.6.0-oss
PyTorch with Intel MKL-DNN - sysstacks/dlrs-pytorch-clearlinux:v0.6.0